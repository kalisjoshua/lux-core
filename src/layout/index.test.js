import React from 'react';
import renderer from 'react-test-renderer';

import Layout from './index';

describe('Default layout - layout', function () {
  const component = renderer.create(
    <Layout />
  );

  it('should match the snapshot', function () {
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
