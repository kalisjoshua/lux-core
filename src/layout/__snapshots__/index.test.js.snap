exports[`Default layout - layout should match the snapshot 1`] = `
<div
  className="page">
  <header
    className="main-header"
    role="banner" />
  <main
    className="body-divider pbl"
    role="main">
    <p>
      Default Page Render
    </p>
  </main>
  <footer
    role="contentinfo">
    <p>
      Â© 
      2016
       
      <a
        className="link--plain"
        href="http://quickenloans.com">
        Quicken Loans
      </a>
    </p>
     
  </footer>
</div>
`;
